### PR TITLE
Feat/new button submit

### PR DESCRIPTION
**Pull request by dominicfernandezvergara**
> **test por ahora**

### CODE REVIEW

File: src/component/homeWeathers/homeWeather.jsx [hunk 1]
```diff
@@ -30,28 +33,28 @@ const HomeWeather = () => {
...
 
   useEffect(() => {
-    if (navigator.geolocation) {
-      navigator.geolocation.getCurrentPosition(
-        handleGeoSuccess,
-        handleGeoError
-      );
-    }
+    dispatch(getWheaterDataCityName());
+    // if (navigator.geolocation) {
```
Comment (javierjah) [inline src/component/homeWeathers/homeWeather.jsx:45]: Please remove comments 😄

File: src/notes/gitHub [hunk 2]
```diff
@@ -1,6 +1,19 @@
-how to Update gitHub
```
Comment (javierjah) [inline src/notes/gitHub:1]: I think this file should be in the .gitignore 🤔

File: src/notes/heroku [hunk 3]
```diff
@@ -4,3 +4,4 @@ gst   (by see status app)
 git add .
```
Comment (javierjah) [inline src/notes/heroku:4]: Same here. Add this file to `.gitignore` file

File: src/component/homeWeathers/homeWeather.jsx (hunk 1 above)
Comment (javierjah) [inline src/component/homeWeathers/homeWeather.jsx:45]: Please remove comments 😄

File: src/pages/home/home.jsx [hunk 4]
```diff
@@ -7,6 +7,7 @@ import ImageShopping from "../../images/undraw_add_to_cart_vkjp.svg";
 // import ImageToDo from "../../images/undraw_following_q0cr.svg";
```
Comment (javierjah) [inline src/pages/home/home.jsx:7]: is always better to remove comments if you don't have words explaining why you have those lines commented 🚀

File: src/redux/weatherStore.jsx [hunk 5]
```diff
@@ -34,25 +39,56 @@ export default function weatherReducer(state = initialData, action) {
...
   dispatch({
     type: GET_WEATHER,
   });
 
   try {
     const API_KEY = "29958692948bbd2bd99b11c5268abf11";
+    // const city = "berlin";
+
     const API_URL = `https://api.openweathermap.org/data/2.5/weather?lat=${latitude}&lon=${longitude}&appid=${API_KEY}&units=metric`;
+    // const API_URL = `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${API_KEY}&units=metric`;
```
Comment (javierjah) [inline src/redux/weatherStore.jsx:91]: do you need this comment??

File: src/redux/weatherStore.jsx [hunk 6]
```diff
@@ -73,3 +109,34 @@ export const getWheaterData = ({ latitude, longitude }) => async (dispatch) => {
...
+  console.log("cityReact", city);
+  dispatch({
+    type: GET_WEATHER_TEST,
+  });
+
+  try {
+    const API_KEY = "29958692948bbd2bd99b11c5268abf11";
+    const citya = city;
+
+    // const API_URL = `https://api.openweathermap.org/data/2.5/weather?lat=${latitude}&lon=${longitude}&appid=${API_KEY}&units=metric`;
```
Comment (javierjah) [inline src/redux/weatherStore.jsx:123]: same here. Do you need this commented url?

**Review by javierjah — CHANGES_REQUESTED** [review]
Great work!

File: src/notes/gitHub (hunk 2 above)
Comment (dominicfernandezvergara) [reply src/notes/gitHub:1]: ok

**Review by javierjah — APPROVED** [review]
Awesomne. totally approved 🚀

**Pull request merged**